src/pages/mc-defaults/index.js

```diff
@@ -24,185 +23,173 @@ import { MultiCurrencyRepository } from 'src/repositories/MultiCurrencyRepositor
-    return(
-        <>
-            <Box
+  return (
+    <>
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          height: '100%',
+          marginTop: '10px'
+        }}
+      >
+        <Grid container spacing={5} sx={{ pl: '10px' }} lg={4} md={7} sm={7} xs={12}>
+          <Grid item xs={12}>
+            <ResourceComboBox
+              endpointId={MultiCurrencyRepository.RateType.qry}
+              name='mc_defaultRTSA'
+              label={_labels.mc_defaultRTSA}
+              valueField='recordId'
+              displayField='name'
+              values={formik.values}
+              onChange={(event, newValue) => {
+                if (newValue) {
+                  formik.setFieldValue('mc_defaultRTSA', newValue?.recordId)
+                } else {
+                  formik.setFieldValue('mc_defaultRTSA', '')
+                }
+              }}
+              error={formik.touched.mc_defaultRTSA && Boolean(formik.errors.mc_defaultRTSA)}
+            />
+          </Grid>
+          <Grid item xs={12}>
+            <ResourceComboBox
+              endpointId={MultiCurrencyRepository.RateType.qry}
+              name='mc_defaultRTPU'
+              label={_labels.mc_defaultRTPU}
+              valueField='recordId'
+              displayField='name'
+              values={formik.values}
+              onChange={(event, newValue) => {
+                if (newValue) {
+                  formik.setFieldValue('mc_defaultRTPU', newValue?.recordId)
+                } else {
+                  formik.setFieldValue('mc_defaultRTPU', '')
+                }
+              }}
+              error={formik.touched.mc_defaultRTPU && Boolean(formik.errors.mc_defaultRTPU)}
+            />
+          </Grid>
+          <Grid item xs={12}>
+            <ResourceComboBox
+              endpointId={MultiCurrencyRepository.RateType.qry}
+              name='mc_defaultRTMF'
+              label={_labels.mc_defaultRTMF}
+              valueField='recordId'
+              displayField='name'
+              values={formik.values}
+              onChange={(event, newValue) => {
+                if (newValue) {
+                  formik.setFieldValue('mc_defaultRTMF', newValue?.recordId)
+                } else {
+                  formik.setFieldValue('mc_defaultRTMF', '')
+                }
+              }}
+              error={formik.touched.mc_defaultRTMF && Boolean(formik.errors.mc_defaultRTMF)}
+            />
+          </Grid>
+          <Grid item xs={12}>
+            <ResourceComboBox
+              endpointId={MultiCurrencyRepository.RateType.qry}
+              name='mc_defaultRTFI'
+              label={_labels.mc_defaultRTFI}
+              valueField='recordId'
+              displayField='name'
+              values={formik.values}
+              onChange={(event, newValue) => {
+                if (newValue) {
+                  formik.setFieldValue('mc_defaultRTFI', newValue?.recordId)
```

Comment:
you can replace   formik.setFieldValue('mc_defaultRTFI', newValue?.recordId || '')